packages/qdrant-loader/src/qdrant_loader/connectors/jira/config.py

```
@@ -149,3 +260,107 @@ def validate_list_items(cls, v: list[str]) -> list[str]:
+                pass
+
+            # Parse relative dates like "-2 days", "-48h", "-2d", "-1w"
+            match = re.match(
+                r"^-(\d+)\s*(days?|hours?|h|d|w|weeks?)$", v.strip(), re.IGNORECASE
+            )
+            if match:
+                amount = int(match.group(1))
+                unit = match.group(2).lower()
+
+                if unit in ("day", "days", "d"):
+                    return datetime.now() - timedelta(days=amount)
```

Ruff (0.15.12)

[warning] `datetime.datetime.now()` called without a `tz` argument

(DTZ005)